dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/deduplication/hibernate/PotentialDuplicate.hbm.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd"
+        [<!ENTITY identifiableProperties SYSTEM "classpath://org/hisp/dhis/common/identifiableProperties.hbm">]
+        >
+
+<hibernate-mapping>
+    <class name="org.hisp.dhis.deduplication.PotentialDuplicate" table="potentialduplicate">
+
+        <cache usage="read-write" />
+
+        <id name="id" column="potentialduplicateid">
+            <generator class="sequence">
+                <param name="sequence">potentialduplicatesequence</param>
+            </generator>
+        </id>
+
+
+        <!-- IdentifiableObject Properties -->
```

Comment:
any reason why this is explicitly specified rather than simply referencing the common identifiableProperties.hbm?

Reply:
Yeah, I don't want the code column :)

Reply:
If thats the case then you can remove Line 5 from this hbm. Its the external entity reference for the identifiableProperties.hbm file.